chain-impl-mockchain/src/vote/payload.rs

```diff
@@ -49,6 +67,15 @@ impl Payload {
 
         match self {
             Self::Public { choice } => bb.u8(payload_type as u8).u8(choice.as_byte()),
+            Self::Private { encrypted_vote } => {
+                let size = encrypted_vote.len();
+                let mut bb = bb.u64(size as u64);
+                for ct in encrypted_vote.iter() {
```

Comment:
Is there a reason for having multiple votes in a single payload?

Reply:
No, it is not that there are multiple votes in the payload. But the encrypted vote is itself a list of some components.

Reply:
Ah, understood, I remembered that. Then it is better to make this less confusing, like having a named field called `encrypted_vote_parts` or something like that.

Reply:
Well, the type itself mask that already I think.